circuits/cpp/src/aztec3/circuits/kernel/private/.test.cpp

```diff
@@ -284,12 +287,20 @@ PrivateInputs<NT> do_private_call_get_kernel_inputs(bool const is_constructor,
     //***************************************************************************
     Composer private_circuit_composer = Composer("../barretenberg/cpp/srs_db/ignition");
 
+    // TODO(Suyash): Oracle needs a bn254::fr private key so we are temporarily casting a secp256k1::fr
+    // private key to bn254::fr.
```

Comment:
Sorry about this hacked-together oracle for testing purposes.

Reply:
No problem!